app/helpers/countries_helper.rb

```diff
@@ -1,9 +1,10 @@
 module CountriesHelper
-  def if_country_get_flag(country_name)
-    name = country_name.downcase.gsub(' ', '-')
-    flag = FileTest.exist?("app/assets/images/flags/#{name}.svg")
+  def if_country_get_flag(country_iso3)
```

Comment:
I think you forgot to pass the iso_3 to the method in the view! Otherwise looks good 👍